src/lib/breadcrumb.tsx

```
@@ -1,72 +1,79 @@
-const ActiveElement = styled(Content)`
-  color: ${({ theme }) => theme.klerosUIComponentsPrimaryText};
-`;
+import { cn } from "../utils";
+import { Button } from "react-aria-components";
+import { clsx } from "clsx";
 
 interface BreadcrumbProps {
   items: { text: string; value: any }[];
-  // eslint-disable-next-line @typescript-eslint/ban-types
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-function-type
   callback?: Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)